bin/warnhist

```diff
@@ -449,7 +662,31 @@ while (<>) {
 #    print STDERR "warning = \"$warning\"\n";
 }
 
-print "Total unique [non-ignored] warnings: $totalcount\n";
+# Sanity check compiled source files with multiple paths when computing
```

Comment:
Try to make certain that the # of compiles of source files with same name matches the number of times files with that name are found in the source directory hierarchy(s).